src/interfaces/store-interfaces/state-interface.ts

```diff
@@ -1,4 +1,8 @@
+import { CertifyIF } from '../stepper-interfaces/certify-interface'
+
 // State model example
 export interface StateModelIF {
-  stateText: string | undefined
+  stateText: string | undefined,
+  currentDate: string,
+  certifyState: CertifyIF
```

Comment:
We need to discuss about how each step will update the state.If each step individually updates the state, we may not need anything additional in step 5 because it is a read only view of what is present in other steps. But this needs to be discussed and refactored if needed

Reply:
OK.

As I understand it, step 5 will have several sections to prompt for additional data, so I guess those will need state and validation as well.